src/views/mfa-verify/WindowsHelloErrorMessageView.js

```diff
@@ -18,14 +18,22 @@ define([
 
     var template = '\
       <span class="icon error-24"></span>\
-      <h4><strong>{{i18n code="enroll.windowsHello.error.notWindows" bundle="login"}}</strong></h4>\
+      <h4><strong>{{{message}}}</strong></h4>\
```

Comment:
Why do you need the unescaped variant? And if it's necessary, we should explicitly add a comment describing why.